indispensable/src/commonMain/kotlin/at/asitplus/signum/indispensable/asn1/Asn1Encoding.kt

```diff
@@ -2,24 +2,18 @@ package at.asitplus.signum.indispensable.asn1
 
 import at.asitplus.KmmResult
 import at.asitplus.catching
-import at.asitplus.signum.indispensable.asn1.BERTags.ASN1_NULL
-import at.asitplus.signum.indispensable.asn1.BERTags.BIT_STRING
-import at.asitplus.signum.indispensable.asn1.BERTags.BOOLEAN
-import at.asitplus.signum.indispensable.asn1.BERTags.GENERALIZED_TIME
-import at.asitplus.signum.indispensable.asn1.BERTags.INTEGER
-import at.asitplus.signum.indispensable.asn1.BERTags.UTC_TIME
-import at.asitplus.signum.indispensable.asn1.DERTags.toExplicitTag
 import at.asitplus.signum.indispensable.io.BitSet
 import com.ionspin.kotlin.bignum.integer.BigInteger
 import com.ionspin.kotlin.bignum.integer.util.toTwosComplementByteArray
 import kotlinx.datetime.Instant
+import kotlin.experimental.or
```

Comment:
are we sure this is the right `or`? being in `experimental` sounds suspicious

Reply:
it is the only one there is for unsigned types